src/boot/bootstrapper-router.tsx

```diff
@@ -38,42 +32,42 @@ const ContextBridge: FC = () => {
 	return null;
 };
 
-const StandaloneListener: FC = () => {
+const StandaloneListener = (): null => {
 	const { route } = useParams<{ route?: string }>();
 	useEffect(() => {
 		if (route) useAppStore.setState({ standalone: route });
 	}, [route]);
 	return null;
 };
 
-const DefaultViewsRegister: FC<{ t: TFunction }> = ({ t }) => {
+type DefaultViewsRegisterProps = { t: TFunction };
+
+const DefaultViewsRegister = ({ t }: DefaultViewsRegisterProps): null => {
 	useEffect(() => {
 		registerDefaultViews(t);
 	}, [t]);
 	return null;
 };
 
-const BootstrapperRouter: FC = () => {
+const BootstrapperRouter = (): JSX.Element => {
```

Comment:
If we move `t` inside `DefaultViewsRegister`, we could remove this component and move its content inside `Bootstrapper`. It is only used there and there aren't particular dependencies which create rerenders. What do you think?